CONTRIBUTING.md

```diff
@@ -54,6 +56,37 @@ Please explain why someone else should merge your proposed change:
 * Does my PR make multiple changes and therefore require multiple commits?
 Ideally, each individual change correlates to an individual commit.
 
+## Capitalization
+
+All headings are sentence case.
+We capitalize the following terms:
+
+* Ansible
+* Candlepin
+* Chef
+* Content View
+* Discovery (service)
+* Hammer
+* Insights (Red Hat product)
+* Kickstart
+* Library
```

Comment:
This is strange at least. Library is very common word. Is this a product name @maximiliankolb?

Reply:
I have no idea. This came up during review of "sentence case in headings" by @apinnick 

I can only assume this comes from Web UI and it means the default "Library" lifecycle environment?